packages/mask/src/plugins/Avatar/Services/rss3.ts

```diff
@@ -35,9 +35,13 @@ export async function getNFTAvatarFromRSS(userId: string, address: string) {
         nft = nfts as NFTRSSNode
     }
 
-    const sig_address = web3.eth.accounts.recover(nft.nft.userId, nft.signature)
-    if (!isSameAddress(sig_address, address)) return
-    return nft.nft
+    try {
+        const sig_address = web3.eth.accounts.recover(nft.nft.userId, nft.signature)
+        if (!isSameAddress(sig_address, address)) return
+        return nft.nft
+    } catch {
+        throw new Error('Failed to recover signature, and please check your connection.')
```

Comment:
Just use 'Failed to recover signature.' as the error message. It's nothing about the connection.